### ISSUE TITLE
Graphic chart and new colors

### ISSUE THREAD

**Issue by stefaweb** [mailwatch/MailWatch#878]
@annixos  posted this #877.

![capture d ecran 2017-06-24 a 18 01 20](https://user-images.githubusercontent.com/12907102/27509961-d43f5a8e-5907-11e7-9246-5b9a14003e4a.png)

**Comment by Skywalker-11** [comment]
I don't think we should change the basic colors in 1.2.x. If someone wants to change them they are able to define their own skin.css. Or we provide this as an alternative style file.

**Comment by stefaweb** [comment]
A good question for v2!   😀

This one have some points.

Yellow in logo don't fit new colors.
New "green-blue" in place of yellow don't fit with blue of selected menu.
Violet for virus is too pure.

It's just my opinion.

**Comment by annixos** [comment]
I modified logo ...
http://imgur.com/tfPcSCN
Logo:
http://imgur.com/uPB4f7H

**Comment by shawniverson** [comment]
I think yellow is still a good brand color for MailWatch.  Although eFa uses this color (or close to it), I don't think MailWatch should have to conform to that.   Just my 2 cents.

**Comment by endelwar** [comment]
There won't be major UI changes in MailWatch 1.2, also current colours and logo font matches the ones of MailScanner.
 
For User Interface of MailWatch 2.0 there will be a brainstorm thread when there will be the need for it (spoiler: I like very much [AdminLTE layout](https://adminlte.io/themes/AdminLTE/index2.html))

**Comment by stefaweb** [comment]
I'll ask my son. I know well this kind of frameworks.

But AdminLTE is a "standard".

A review of 22 admin templates.

https://colorlib.com/wp/free-html5-admin-dashboard-templates/

**Comment by annixos** [comment]
https://colorlib.com/polygon/gentelella/
This color scheme best. For me :)

Dne 3. 7. 2017 18:02 napsal uživatel "Stéphane" <notifications@github.com>:

**Comment by stefaweb** [comment]
Soothing !

http://getbootstrap.com
https://angularjs.org
https://loopback.io

**Comment by asuweb** [comment]
There's no reasons people can't change the colours to suit by editing the css in v1.2.  In v2 we could include a colour picker in a settings page to configure it via the GUI.

Personally, I use these colours for mail identification in mailwatch.

![image](https://user-images.githubusercontent.com/13250955/27806543-0a53d7e4-6033-11e7-9ee5-e8fff2c8af20.png)

**Comment by stefaweb** [comment]
Good idea the colour picker.

**Comment by asuweb** [comment]
Good job with the graphs.  They look good.

![image](https://user-images.githubusercontent.com/13250955/28069215-91294d90-6640-11e7-94ac-ead4bb216674.png)
